subjects/bigger/README.md

```diff
@@ -6,11 +6,7 @@ Create the function `bigger` that gets the biggest positive number in the `HashM
 
 ### Notions
 
-[hash maps](https://doc.rust-lang.org/book/ch08-03-hash-maps.html)
-
-### Dependencies
-
-rand = "0.7"
+- [hash maps](https://doc.rust-lang.org/book/ch08-03-hash-maps.html)
 
```

Comment:
why remove the dependencies?
we use the version of the dependencies because of some problems we encountered.
Did it pass without using this version?

Reply:
Yeah, I didn't have to use rand in that exercise, so I thought that it was a mistake. The exercise is to get the biggest positive number in a HashMap, was I supposed to use rand for that?